index.html

```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
-                        <div class="pa-social-section">
-                            <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Connect</strong> with us</h1>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-facebook"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-behance"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-dribbble"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-instagram"></i></span></a>
-                            <p class="pa-p-v1 pa-white pa-contact-lastline fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay03Duration10">BrainChild Services</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10">Cochin,Kerala</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">mail</i></span>contact@brainchildservices.com</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">location_on</i></span>Google Map</p>
+                        <div>
+                            
+                                <iframe width="400" height="450"
+                                    src="https://maps.google.com/maps?q=chottanikkara,Kerala&t=k&z=11&ie=UTF8&iwloc=&output=embed"
+                                    frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe><a
+                                    href="https://embedgooglemap.net/maps/36"></a><br>
+                                    </div>
                     </div>
-                </div>
+                    </div>
+                
                 <!-- Contact Form -->
                 <div class="col-12 col-md-6 col-xl-5 pa-contact-form d-flex justify-content-end">
                     <div class="pa-glass pa-right">
-                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10">Send us a <strong>Message</strong></h1>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Add your Contact details here and send us a message </strong></h1>
 
```

Comment:
after this content give a proper space from the form.
![image](https://user-images.githubusercontent.com/29180114/126025848-b6fed980-958b-4db3-8e78-a29ca35236c7.png)

Reply:
done